.github/CODEOWNERS

```diff
@@ -3,7 +3,7 @@
 
 # These owners will be the default owners for everything in the repo.
 #*       @defunkt
-*       @MatthewPyle-NOAA @ShunLiu-NOAA @JacobCarley-NOAA @chan-hoo @BenjaminBlake-NOAA @RatkoVasic-NOAA @JiliDong-NOAA @hu5970
+*       @RRFS-devs
```

Comment:
@MatthewPyle-NOAA, I've updated the codeowners, but it still says "unknown owner".

Reply:
Yeah, I saw that.  Is there a need for the codeowners file since we have github level control of the team?  Can you see this list of developers (which should be able to issue and interact with PRs and other normal non-admin github activity) [https://github.com/orgs/NOAA-EMC/teams/rrfs-devs](https://github.com/orgs/NOAA-EMC/teams/rrfs-devs)